hyperdrive_test.go

```diff
@@ -51,6 +51,11 @@ var _ = Describe("Hyperdrive", func() {
 				ipChans := make([]chan Object, entry.numHyperdrives)
```

Comment:
Each of these `ipChans` should also be buffered with a capacity of `2 * (entry.numHyperdrives + 1) * int(entry.maxHeight)`.